Facebook.Unity.Editor/iOS/FixupFiles.cs

```diff
@@ -53,7 +53,12 @@ public static void AddBuildFlag(string path)
             string projPath = Path.Combine(path, Path.Combine("Unity-iPhone.xcodeproj", "project.pbxproj"));
             PBXProject proj = new PBXProject();
             proj.ReadFromString(File.ReadAllText(projPath));
-            string targetGUID = proj.TargetGuidByName("Unity-iPhone");
+
+            // When building with Unity 2019.3 and later versions, there are two Xcode targets generated.
+            // The UnityFramework target will include the code and Unity-iPhone is only a thin wrapper around it.
+            // With earlier versions everything is in a single Unity-iPhone target.
+            var targetName = GetUnityVersionNumber() >= 201930 ? "UnityFramework" : "Unity-iPhone";
+            string targetGUID = proj.TargetGuidByName(targetName);
```

Comment:
Will this better?

```C#
#if UNITY_2019_3_OR_NEWER
		string targetGUID = proj.GetUnityMainTargetGuid();
#else
		string targetName = PBXProject.GetUnityTargetName();
		string targetGUID = proj.TargetGuidByName(targetName);
#endif
```

Reply:
No, that won't work. We cannot use preprocessor directives here as this file is compiled into a DLL that has to support various Unity versions.

Reply:
@Nezz I see but still didn't `GetUnityFrameworkTargetGuid` and `GetUnityTargetName` better than constant?

Reply:
`GetUnityFrameworkTargetGuid` is only available in Unity 2019.3 and later. If you include this DLL in a 2018.4 project, you will get a compiler error.

`GetUnityTargetName` was removed in Unity 2019.3. If you include this DLL in a 2019.3 project, you will get a compiler error.

Reply:
Thanks you for clarification